pages/_app.tsx

```diff
@@ -15,6 +16,13 @@ class OdysseyApp extends App {
     return { pageProps };
   }
 
+  componentDidMount() {
+    // Let's just hardcode the hotjar
+    if (process.env.NODE_ENV === "production") {
```

Comment:
added a check for `NODE_ENV === 'production'` to make sure that dev work doesn't pollute our hotjar data